shared/scripts/pktgen_perf.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+# usage sh pktgen.sh $dst $device $queues %size
+
+DST=$1
+NET_DEVICE=$2
+QUEUES=$3
+SIZE=$4
+
+lsmod | grep pktgen || modprobe  pktgen
+echo reset > /proc/net/pktgen/pgctrl
+ifconfig $NET_DEVICE up
+
+function pgset() {
+    local result
+
+    echo $1 > $PGDEV
+
+    result=`cat $PGDEV | fgrep "Result: OK:"`
+    if [ "$result" = "" ]; then
+         cat $PGDEV | fgrep Result:
+    fi
+}
+
+function pg() {
+    echo inject > $PGDEV
+    cat $PGDEV
+}
+
```

Comment:
This function isn't called in the script.

Reply:
yes, it's not be called. the interest thing is all the examples I could found from web are all included function pg, example https://github.com/danieltt/pktgen/tree/master/examples , and also pktgen.sh in avocado-vt, so I think let's just keep this in this script. thx

Reply:
If there is no error when remove this function, I think it's better to remove it.